### ISSUE TITLE
[Feature] App Group Panel with list view mode

### ISSUE THREAD

**Issue by roke-julian-lockhart** [FireCubeStudios/StartStrikesBack#2]
Per https://github.com/FireCubeStudios/StartStrikesBack/blob/a1281167860f272c81106ea5443aef4f1587039d/README.md#preview-:

> ![test](https://camo.githubusercontent.com/19134f8240a43fe75d8f7d7c5c36dd9ed043a23a0fb392d22d721f7a345cfc25/68747470733a2f2f6d656469612e646973636f72646170702e6e65742f6174746163686d656e74732f313037383031313537323030373437373239382f313139383830323732343433353830383332362f696d6167652e706e673f65783d36356330336230612669733d363561646336306126686d3d64353166646661663862656565613337623539396632633234353565623035663632353138363765636664656164373466393634383562386330626631396334263d26666f726d61743d77656270267175616c6974793d6c6f73736c6573732677696474683d31333138266865696768743d31303238)

...I expect that this software shall (by default, at least) utilize a grid to display favourite applications. However, I have no reason to use a grid - the application icons are of no importance to me. Use KDE's Application Launcher as an example:

> ![20240126T180655GMT](https://github.com/FireCubeStudios/StartStrikesBack/assets/42837531/dd93ea60-c7ca-453c-b285-cd1ec6c22ac4) ![20240126T180546GMT](https://github.com/FireCubeStudios/StartStrikesBack/assets/42837531/0bc3449e-b503-4da4-b72b-8aa27d4968c1)

...versus:

> ![20240126T180805GMT](https://github.com/FireCubeStudios/StartStrikesBack/assets/42837531/7950a0a9-0e95-4037-83db-c11aab9f7d1e) ![20240126T180801GMT](https://github.com/FireCubeStudios/StartStrikesBack/assets/42837531/8afe103f-f5ad-476d-b8f7-72af71b6e24a)

Really, I'd like a tree view though, since I want to be able to view the content of folders inline, like the Windows 7 & 10 Start Menus used to allow.

**Comment by FireCubeStudios** [comment]
Yes this is easily possible by either creating a custom shell or by adding a new app panel supporting listview. I guess I can make a listview style panel
